common/source/docs/common-tarot-gimbal.rst

```diff
@@ -95,11 +95,11 @@ If using the Mission Planner a Camera Gimbal setup screen is available as shown
 
 If manually setting parameters:
 
-- :ref:`MNT_TYPE <MNT_TYPE>` = 1 (for "Servo" gimbal)
-- :ref:`MNT_RC_IN_TILT <MNT_RC_IN_TILT>` = 6 to specify that RC input channel 6 (normally the tuning knob) will be used to control the pitch/tilt of the gimbal
-- :ref:`MNT_ANGMAX_TIL <MNT_ANGMAX_TIL>` = 0 to specify the gimbal's highest angle is zero degrees (i.e. forward)
-- :ref:`MNT_ANGMIN_TIL <MNT_ANGMIN_TIL>` = -9000 to specify the gimbal's lowest angle is 90 degrees (i.e. down)
-- :ref:`MNT_STAB_TILT <MNT_STAB_TILT>` = 0 to specify ArduPilot should not send tilt compensation to the gimbal (the gimbal does this by itself)
+- :ref:`MNT1_TYPE <MNT1_TYPE>` = 1 (for "Servo" gimbal)
+- :ref:`RC6_OPTION <RC6_OPTION>` = 213 to specify that RC input channel 6 (normally the tuning knob) will be used to control the pitch/tilt of the gimbal
+- :ref:`MNT1_PITCH_MAX <MNT1_PITCH_MAX>` = 0 to specify the gimbal's highest angle is zero degrees (i.e. forward)
+- :ref:`MNT1_PITCH_MIN <MNT1_PITCH_MIN>` = -90 to specify the gimbal's lowest angle is 90 degrees (i.e. down)
+- :ref:`RCx_OPTION <RC1_OPTION>` = 0 to specify ArduPilot should not send tilt compensation to the gimbal (the gimbal does this by itself)
```

Comment:
```suggestion
```
there are no longer STAB options

Reply:
Fixed

Reply:
still is there apparently

Reply:
Had it sitting in my staged. Should be there now.